lib/models/mirror.js

```diff
@@ -3,6 +3,7 @@
 const errors = require('storj-service-error-types');
 const mongoose = require('mongoose');
 const SchemaOptions = require('../options');
+const _ = require('lodash');
```

Comment:
Part of the reason I had thought to have this be a separate PR,  since I don't think this is related to billing, but if you want to include with this one please just coordinate within this one.